docs/sources/tempo/getting-started/docker-example.md

```diff
@@ -31,6 +44,26 @@ To follow this guide, you need:
 - Docker
 - The docker-compose plugin
 
+{{< admonition type="tip" >}}
+Alternatively, you can try out this example in our interactive learning environment: [Quick start for Tempo](https://killercoda.com/grafana-labs/course/tempo/quick-start).
+
+It's a fully configured environment with all the dependencies already installed.
+
+![Interactive](/media/docs/tempo/tempo-ile.svg)
+
+Provide feedback, report bugs, and raise issues in the [Grafana Killercoda repository](https://github.com/grafana/killercoda).
+{{< /admonition >}}
+
+<!-- INTERACTIVE ignore END -->
+
+{{<docs/ignore>}}
+> **Note**: A setup script is running in the background to install the necessary dependencies. This should take no longer than 30 seconds. Your instance will be ready to use once you `Setup complete. You may now begin the tutorial`.
+{{</docs/ignore>}}
```

Comment:
Normally we use and admonition shortcode. Do we need to remove any shortcodes when we use markdown for Killercoda and Hugo?

Reply:
We have support for some shortcodes in Killercoda but not all of them. Admonitions are not yet supported so we write Killercoda notes in plain Markdown. I've created the documentation in https://github.com/grafana/killercoda/pull/118.

Reply:
https://github.com/grafana/killercoda/pull/121 🎉

Reply:
tested and works perfectly: https://killercoda.com/grafana-dev-testing/course/tempo/quick-start